lib/translator/inquiry.h

```diff
@@ -25,6 +25,12 @@
 
 namespace translator {
 
+// constants for IdentifierLength field of DeviceIdentificationVPD
+constexpr uint8_t kIdentifierLengthEui64 = 0x8;
+constexpr uint8_t kIdentifierLengthNguid = 0x10;
+
+StatusCode RawToScsiCommand(absl::Span<const uint8_t>,
```

Comment:
nit: other places have been including variable names in the .h declarations. Although they're not necessary, as @JustinAWei  said, they could provide helpful context esp when the type is a `absl::Span<const uint8_t>`